timestamp,category,user_id,function,line_number,flag,message,exception
2024-06-09 21:55:02,ERROR,asif24,create_or_pass_folder,199,0,Function create_or_pass_folder error,"Function create_or_pass_folder error
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\app.py"", line 192, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:f6406113-f01e-002c-4489-baf73b000000
Time:2024-06-09T16:25:01.8911641Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:f6406113-f01e-002c-4489-baf73b000000
Time:2024-06-09T16:25:01.8911641Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error>"
2024-06-09 21:55:02,ERROR,asif24,create_or_pass_folder,208,0,"Function create_or_pass_folder Error creating folder'asif24': Value for one of the query parameters specified in the request URI is invalid.
RequestId:f6406113-f01e-002c-4489-baf73b000000
Time:2024-06-09T16:25:01.8911641Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:f6406113-f01e-002c-4489-baf73b000000
Time:2024-06-09T16:25:01.8911641Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error> ","Function create_or_pass_folder Error creating folder'asif24': Value for one of the query parameters specified in the request URI is invalid.
RequestId:f6406113-f01e-002c-4489-baf73b000000
Time:2024-06-09T16:25:01.8911641Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:f6406113-f01e-002c-4489-baf73b000000
Time:2024-06-09T16:25:01.8911641Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error> 
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\app.py"", line 192, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:f6406113-f01e-002c-4489-baf73b000000
Time:2024-06-09T16:25:01.8911641Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:f6406113-f01e-002c-4489-baf73b000000
Time:2024-06-09T16:25:01.8911641Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error>"
2024-06-09 21:55:02,INFO,asif24,home,1408,1,User asif24 logged in successfully,
2024-06-09 21:55:03,INFO,asif24,update_bar_chart_from_blob,319,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-09 21:55:26,INFO,asif24,update_bar_chart_from_blob,319,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-09 21:55:29,INFO,asif24,get_data_source,2042,1,table_update route successfully send data,
2024-06-09 21:56:18,ERROR,asif24,delete_files,1999,0,delete for delete route: draft not found,"delete for delete route: draft not found
NoneType: None"
2024-06-09 21:56:19,INFO,asif24,get_data_source,2042,1,table_update route successfully send data,
2024-06-09 21:57:06,ERROR,asif24,delete_files,1999,0,delete for delete route: draft not found,"delete for delete route: draft not found
NoneType: None"
2024-06-09 21:57:06,INFO,asif24,get_data_source,2042,1,table_update route successfully send data,
2024-06-09 21:57:37,ERROR,asif24,delete_files,1999,0,delete for delete route: reg.pdf not found,"delete for delete route: reg.pdf not found
NoneType: None"
2024-06-09 21:57:37,INFO,asif24,update_bar_chart_from_blob,319,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-09 21:57:37,INFO,asif24,delete_files,2004,1,Selected vault files deleted successfully,
2024-06-09 21:57:49,INFO,asif24,update_bar_chart_from_blob,319,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-09 21:57:51,INFO,asif24,get_data_source,2042,1,table_update route successfully send data,
2024-06-09 21:58:07,INFO,asif24,get_data_source,2042,1,table_update route successfully send data,
2024-06-09 21:58:28,INFO,asif24,Cogni_button,1722,1,CogniLink load button pressed,
2024-06-09 22:02:05,INFO,asif24,update_bar_chart_from_blob,319,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-09 22:02:07,INFO,asif24,update_when_file_delete,607,1,Function update_when_file_delete Data Loaded Successfully,
2024-06-09 22:02:07,INFO,asif24,Cogni_button,1734,1,Load Data Function executed in 219.02317428588867 seconds,
2024-06-09 22:02:09,INFO,asif24,get_data_source,2042,1,table_update route successfully send data,
2024-06-09 22:02:47,ERROR,asif24,delete_files,1999,0,delete for delete route: draft not found,"delete for delete route: draft not found
NoneType: None"
2024-06-09 22:03:42,INFO,asif24,get_data_source,2042,1,table_update route successfully send data,
